src/main/java/mate/academy/controller/AuthenticationController.java

```diff
@@ -27,4 +36,16 @@ public UserResponseDto register(@RequestBody @Valid UserRegistrationDto userRequ
                 userRequestDto.getPassword());
         return userMapper.mapToDto(user);
     }
+
+    @PostMapping("/login")
+    public ResponseEntity<Object> login(@RequestBody @Valid UserLoginDto userLoginDto)
+            throws AuthenticationException {
+        User user = authenticationService
+                .login(userLoginDto.getLogin(), userLoginDto.getPassword());
+        String token = jwtTokenProvider.createToken(userLoginDto.getLogin(), user.getRoles()
```

Comment:
```suggestion
        String token = jwtTokenProvider.createToken(user.getEmail(), user.getRoles()
```
better use user property